modules/apps/asset/asset-categories-admin-web/src/main/resources/META-INF/resources/js/ActionsComponentPropsTransformer.js

```diff
@@ -28,23 +28,25 @@ const ACTIONS = {
 			buttonAddLabel: Liferay.Language.get('delete'),
 			multiple: true,
 			onSelect: (selectedItems) => {
-				if (selectedItems) {
+				if (selectedItems.length) {
 					if (
 						confirm(
 							Liferay.Language.get(
 								'are-you-sure-you-want-to-delete-the-selected-entries'
 							)
 						)
 					) {
-						selectedItems.forEach((item) => {
-							vocabulariesForm.appendChild(item.cloneNode(true));
-						});
+						const input = document.createElement('input');
+
+						input.name = `${portletNamespace}rowIds`;
+						input.value = selectedItems.map((item) => item.value);
+
+						vocabulariesForm.appendChild(input);
```

Comment:
Yea, the input stuff is super weird, but an easy bandaid if we remember to address it soon.